.muzak.exs

```diff
@@ -0,0 +1,117 @@
+# Reset the tmp_dir, Fake data ETS table and block_addresses before each run.
+before_suite = fn ->
+  tmp_dir = Application.get_env(:hexpm, :tmp_dir)
+  File.rm_rf(tmp_dir)
+  File.mkdir_p(tmp_dir)
+
+  Hexpm.Fake.reset()
+  Ecto.Adapters.SQL.Sandbox.mode(Hexpm.RepoBase, :auto)
+  Hexpm.BlockAddress.reload()
+  Ecto.Adapters.SQL.Sandbox.mode(Hexpm.RepoBase, :manual)
+  :ok
+end
+
+mutators = [
+  Muzak.Mutators.Constants.Atoms,
+  Muzak.Mutators.Constants.Booleans,
+  Muzak.Mutators.Constants.Lists,
+  Muzak.Mutators.Constants.Numbers,
+  Muzak.Mutators.Constants.Strings,
+  Muzak.Mutators.Conditionals.Boundary,
+  Muzak.Mutators.Conditionals.Replace,
+  Muzak.Mutators.Conditionals.Strict,
+  Muzak.Mutators.Functions.Rename
+]
+
+exclude_files = fn files ->
+  files_to_exclude = ["lib/hexpm/fake.ex"]
+  Enum.reject(files, fn {path, _} -> path in files_to_exclude end)
+end
+
+%{
+  default: [
+    nodes: 1,
+    mutators: mutators,
+    before_suite: before_suite,
+    min_coverage: 80,
```

Comment:
If the mutation tests end up with more than 20% of survivors, muzak will exit with a non-zero code. We can play with this to our taste.

Reply:
And that's just for the default profile - the current setting for the CI profile is 90% success, so it will fail if more than 10% of mutations aren't caught. Also, you can have as many configuration profiles as you want/need with any combination of these settings.